Release 0.0.28 (#91)
Regression-Games/RegressionDocs: docs/changelog.mdx

# Changelog

## Release 0.0.28

_September 23, 2024_

**New Features**

- Bot Sequences can now be created, edited, and deleted directly from the in-game overlay
- Bot segments that rely on image sources can now load images from paths vs just raw data
- Direct login is no longer supported for the SDK; API keys are now required
- Bot Sequences are now reloaded automatically from disk when played
- Object detection bot segments can now define a confidence threshold (text queries only)

**Bug Fixes**

- Improved language and explanations within the website
- Properly handle failing behaviour segments that dont load or timeout

> - Use of computer-vision capabilities now requires authentication

Reviewer: `now requires authentication` -> is fine, but we should mention that this explicitly requires using API keys / having an API key setup in the SDK

Author: Added

- Sprite rendering within the SDK has been improved
- Fixed a bug where the overlay would throw a null exception when opening for the first time

> - Fixed issues where the game would hang when starting

Reviewer: ```suggestion
- Fixed issues causing the game to freeze for a few seconds when starting
```

> - Data from the end of a segment is now properly saved

Reviewer: ```suggestion
- Data for the last segment of a recording is now properly saved
```
